src/libAtomVM/term.h

```diff
@@ -1554,4 +1554,14 @@ static inline int term_find_map_pos(Context *ctx, term map, term key)
     return -1;
 }
 
+static inline term term_find_map_value(Context *ctx, term map, term key, term default_value)
```

Comment:
I think we should move this function to `interop.h` if its main purpose is helping in writing C components.

Reply:
Sure.